02_julia_intro/02_julia_intro.jl

```diff
@@ -663,8 +706,21 @@ Now that the most important details of syntax have been discussed, let's focus o
 # ╔═╡ d1ae60a8-544e-11eb-15b5-97188dc41aa8
 md"
 # **Julia's Ecosystem**: Basic plotting and manipulation of DataFrames
```

Comment:
Now that the most important details of syntax have been discussed, let's focus our attention on some packages of Julia's ecosystem.

Now that we have discussed the most important details of Julia's syntax, let's focus our attention on some of the packages in Julia's ecosystem.